src/common.jl

```diff
@@ -71,6 +71,19 @@ function extend_limits(vec, limits)
     (limits == (0.,0.) || limits == [0.,0.]) ? plotting_range_narrow(mi, ma) : (mi, ma)
 end
 
+sort_by_keys(dict::Dict) = collect(sort(collect(dict), by=x->x[1]))
+
+function sorted_keys_values(dict::Dict; k2s=true)
```

Comment:
Just a question: is this sorting an expected operation for barplot and boxplot? Is it because dictionaries are not ordered?

Reply:
I added sorting, because iterating over test dicts in https://github.com/Evizero/UnicodePlots.jl/blob/326a4eb04279548891928cbb92868e0c51f0cf27/test/tst_barplot.jl#L20 has different behavior between `1.0` and `1.6` causing the comparison against reference files to fail in `1.0`.

I don't know if there is a better solution ?

Reply:
It seems doesn't change the overall behavior of `barplot`, i.e., :

```julia
barplot(["A", "B"], [1, 2])
barplot(["B", "A"], [2, 1])
```

generates different results.

Reply:
That's expected, it doesn't not use `Dict`s here.